### PR TITLE
feature/485 allow multiple modules within a course

### PR DESCRIPTION
**Pull request by superafroman**
Updates so learning record supports modules. And fixed used of TERMINATED following on from video updates.

### CODE REVIEW

File: src/lib/xapi.ts [hunk 1]
```diff
@@ -252,27 +254,28 @@ export async function record(
...
+				}
+				break
+			case 'link':
+				type = Type.Link
+				break
+			case 'video':
+				type = Type.Video
+				break
+			default:
+				throw new Error(`Unknown course type ${course.getType()}`)
```
Comment (tav) [inline src/lib/xapi.ts]: `module.type` ?

File: src/ui/component/LearningRecordTable.html [hunk 2]
```diff
@@ -13,7 +13,7 @@
             <td><a href="/courses/{{course.id}}">{{course.title}} </a></td>
             <td>{{i18n(course.getType())}}</td>
             <td>{{course.getDuration() || '-'}}</td>
-            <td>{{formatDate(course.record.completionDate)}}</td>
+            <td>{{formatDate(course.getCompletionDate(signedInUser))}}</td>
```
Comment (tav) [inline src/ui/component/LearningRecordTable.html]: *throw up*

Comment (superafroman) [reply src/ui/component/LearningRecordTable.html]: Incremental improvements.

File: Dockerfile [hunk 3]
```diff
@@ -39,8 +39,8 @@ ENV AUTHENTICATION_SERVICE_URL=http://identity.local.cshr.digital:9443 \
   PORT=3001 \
   SESSION_SECRET=topsecret
 
-# TODO(tav): Create an actual user so that we don't run this as root
```
Comment (tav) [inline Dockerfile]: Leave the note in there until the root issue is addressed?

Comment (superafroman) [reply Dockerfile]: Maybe it's cos I'm a cowboy, but I don't think this is such an issue when running inside containers.

**Pull request merged**